tests/server/test_static_prefix_security.py

```diff
@@ -0,0 +1,24 @@
+from mlflow.server.auth.utils import is_unprotected_route
```

Comment:
Can you add test validation to the auth protection suites instead of creating a new test suite here?